libtenzir/builtins/operators/batch.cpp

```diff
@@ -6,6 +6,8 @@
 // SPDX-FileCopyrightText: (c) 2023 The Tenzir Contributors
 // SPDX-License-Identifier: BSD-3-Clause
 
+#include "tenzir/detail/assert.hpp"
+
```

Comment:
This seems like an IWYU leftover, it should already be included many times over transitively.